Trent edits 20230927 (#82)
jessegmeyerlab/proteomics-tutorial: content/08.enrichment.md

### Peptide enrichment 
* Glycosylation

Mass spectrometry-based analysis of protein glycosylation has emerged as the premier technology to characterize such a universal and diverse class of biomolecules.

> Glycosylation is a heterogenous post-translational modification that decorates many proteins within the proteome, confering broad changes in protein activity. [@DOI: https://doi.org/10.1093/glycob/12.4.43R; https://doi.org/10.1016/j.cell.2006.08.019; https://doi.org/10.1101/9781621824213] 

Reviewer: ```suggestion
Glycosylation is a heterogenous post-translational modification that decorates many proteins within the proteome, conferring broad changes in protein activity. [@DOI:10.1093/glycob/12.4.43R; @DOI:10.1016/j.cell.2006.08.019; @DOI:10.1101/9781621824213] 
```

This PTM can take many forms.
The covalent linkage of mono- or oligosaccharides to polypeptide backbones through a nitrogen atom of asparagine (N) or an oxygen atom of serine (S) or threonine (T) side-chains creates N- and O-glycans, respectively.  

> The heterogenity of proteoglycans is not directly tied to the genome, and thus can't be inferred from gene expression. 

Reviewer: ```suggestion
The heterogenity of proteoglycans is not directly tied to the genome, and thus cannot be inferred. 
```

Rather, the abundance and activity of protein glycosylation is governed by glycosyltransferases and glycosidases which add and remove glycans, respectively. 
The fields of glycobiology and bioanalytical chemistry are intricately intertwined with mass spectrometry at the center thanks in part to its power of detecting any modification that imparts a mass shift. 

> Due to the myriad glycan structures and proteins which harbor them, the enrichment of glycoproteins or glycopeptides is not as streamlined as that of other PTMs [@DOI: https://doi.org/10.1074/mcp.R120.002277]. 

Reviewer: ```suggestion
Due to the myriad glycan structures and proteins which harbor them, the enrichment of glycoproteins or glycopeptides is not as streamlined as that of other PTMs [@DOI:10.1074/mcp.R120.002277]. 
```

The enrichment of glycoproteome from the greater proteome inherently introduces bias prior to the LC-MS/MS analysis.
One must take into account which class or classes of glycopeptides they are interested in analyzing before enrichment for optimal LC-MS/MS results. 
Glycopeptides can be enriched via glycan affinity, for example to glycan-binding proteins, chemical properties like charge or hydrophilicity, chemical coupling of glycans to stationary phases, and by bioorthogonal, chemical biology approaches.
Glycan affinity-based enrichment strategies include the use of lectins, antibodies, inactivated enzymes, immobilized metal affinity chromatography (IMAC), and metal oxide affinity chromatography (MOAC).
The enrichment of glycopeptides by their chemical properties, for example by biopolymer charge and hydrophobicity, include hydrophilic interaction chromatography (HILIC), electrostatic repulsion-hydrophilic interaction chromatography (ERLIC), and porous graphitic carbon (PGC).

> One variation of ERLIC combines strong anion exchange, electrostatic repulsion, and hydrophilic interaction chromatography (SAX-ERLIC) has risen in popularity thanks to robustness and commercially available enrichment kits [@DOI: https://doi.org/10.1007/978-1-0716-1241-5_8].

Reviewer: ```suggestion
One variation of ERLIC combines strong anion exchange, electrostatic repulsion, and hydrophilic interaction chromatography (SAX-ERLIC) has risen in popularity thanks to robustness and commercially available enrichment kits [@DOI:10.1007/978-1-0716-1241-5_8].
```

> Chemical coupling methods most often used to enrich the glycoproteome employ hydrazide chemistry for sialylated glycopeptides.

Reviewer: ```suggestion

Chemical coupling methods most often used to enrich the glycoproteome employ hydrazide chemistry for sialylated glycopeptides.
```

Glycan are cleaved from the stationary phase by PNGase F.
The dependence of chemical coupling methods on PNGase F biases their output toward N-glycopeptides.
Alkoxyamine compounds and boronic acid-based methods have also shown utility. 

> We direct readers to several reviews on glycopeptide enrichment strategies [@DOI: https://doi.org/10.1074/mcp.R120.002277; https://doi.org/10.1016/j.ab.2009.08.023; https://doi.org/10.1016/j.bbapap.2014.05.002; https://doi.org/10.1039/C3RA42960A; https://doi.org/10.1021/acs.analchem.6b04343]

Reviewer: ```suggestion
We direct readers to several reviews on glycopeptide enrichment strategies [@DOI:10.1074/mcp.R120.002277; @DOI:10.1016/j.ab.2009.08.023; @DOI:10.1016/j.bbapap.2014.05.002; @DOI:10.1039/C3RA42960A; @DOI:10.1021/acs.analchem.6b04343]
```

> * SISCAPA

Reviewer: ```suggestion
```

Second, phosphopeptide enrichment is not hindered by small, lipophilic, and very acidic or alkaline proteins [@PMID:22002794]. 
Third, prefractionation techniques such as strong anionic ion-exchange chromatography (SAX), strong cationic ion-exchange chromatography (SCX) and hydrophilic interaction liquid chromatography (HILIC) are easier to use for peptide separation than they are for protein separation, and they are more sensitive than 2D-gel electrophoresis, allowing for the identification of less abundant phosphopeptides [@PMID:15966007]. 
As a result, phosphopeptide enrichment has yielded more experimental data than phosphoprotein enrichment [@PMID:17480206]. 
Phosphopeptide enrichment is typically done after any isobaric labeling strategy, although several have investigated the importance of order at these stages.

Phosphopeptide enrichment often uses titanium dioxide (TiO2) [@PMID:15858219] and/or IMAC such as Fe3+ coupled to solid-phase materials [@PMID:3085541; @PMID:35011700, @PMID:26584922]. 
Often organic acids such as glutamic acid, lactic acid, glycolic acid is added to compete with acidic non-phosphopeptides for binding to the metal-ions. 

> Carr and coworkers even demonstrated phosphoproteome analysis without any enrichment [@DOI:https://doi.org/10.1006/abio.1996.0313].

Reviewer: ```suggestion
Carr and coworkers even demonstrated phosphoproteome analysis without any enrichment [@DOI:10.1006/abio.1996.0313].
```
